### PR TITLE
Added elysium testnet chain

### PR DESCRIPTION
**Pull request by aqeelVaival**
### PR-Codex overview
This PR adds support for the `elysiumTestnet` chain with its definitions and configurations.

### Detailed summary

1. Added `elysiumTestnet` chain definition in `chains/index.ts`
2. Created `elysiumTestnet` chain configurations in `chains/definitions/elysiumTestnet.ts`
3. Utilized `defineChain` function to define the chain with ID, name, currency, RPC URLs, block explorers, and testnet flag

---

## PR-Codex overview
This PR introduces support for the `elysiumTestnet` chain, updating various configuration files and adding necessary exports. It also includes adjustments to package dependencies.

### Detailed summary
- Removed `src/node/trustedSetups_cjs.ts.changeset/funny-dragons-sin.md`.
- Updated `pnpm-lock.yaml` with new chain details.
- Modified `src/package.json` to include keywords.
- Reformatted `src/jsr.json` for better readability.
- Updated `tsconfig.json` to include `elysiumTestnet.ts`.
- Added export for `elysiumTestnet` in `src/chains/index.ts`.
- Refactored `src/node/trustedSetups.ts` for path resolution.
- Created `src/chains/definitions/elysiumTestnet.ts` with chain configuration.

> The following files were skipped due to too many changes: `package-lock.json`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by aqeelVaival** [comment]
@jxom  please help to proceed!

File: package-lock.json
Comment (jxom) [inline package-lock.json:1]: revert

File: pnpm-lock.yaml
Comment (jxom) [inline pnpm-lock.yaml:1]: revert

File: src/jsr.json
Comment (jxom) [inline src/jsr.json:1]: revert

File: src/node/trustedSetups.ts
Comment (jxom) [inline src/node/trustedSetups.ts:1]: revert

File: src/node/trustedSetups_cjs.ts
Comment (jxom) [inline src/node/trustedSetups_cjs.ts:1]: revert

File: src/package.json
Comment (jxom) [inline src/package.json:1]: revert

File: tsconfig.json
Comment (jxom) [inline tsconfig.json:1]: revert

File: src/chains/index.ts [hunk 1]
```diff
@@ -374,6 +374,7 @@ export { zora } from './definitions/zora.js'
 export { zoraSepolia } from './definitions/zoraSepolia.js'
 export { zoraTestnet } from './definitions/zoraTestnet.js'
 export { zircuitTestnet } from './definitions/zircuitTestnet.js'
+export {elysiumTestnet} from './definitions/elysiumTestnet.js';
```
Comment (jxom) [inline src/chains/index.ts:377]: please keep chains in alphabetical order

**Comment by jxom** [comment]
Please address the comments and open as another PR when done.

**Pull request closed without merging**